### ISSUE TITLE
The sound is silent

### ISSUE THREAD

**Issue by ThingUroboros** [mborjesson/PCem-wx-SDL2#8]
The sound is silent could it amplify it or the possibility of manual increase?

Thank you for answer

**Comment by mborjesson** [comment]
I agree that the sound is a bit low, but unfortunately I don't know how to fix that :(
